### ISSUE TITLE
Docker config on controller is pointing on wrong docker registry address

### ISSUE THREAD

**Issue by dchirikov** [clustervision/trinity#635]
It was
`OPTIONS=--insecure-registry 192.168.1.254:5050`
instead of
`OPTIONS=--insecure-registry controller:5050`
in file `/etc/sysconfig/docker`

**Comment by hansthen** [comment]
Unable to reproduce.

**Comment by quentinleburel** [comment]
Benchmark cluster has the same issue

```
[root@controller(BENCHMARK) /]# cat /etc/sysconfig/docker
# /etc/sysconfig/docker

# Modify these options if you want to change the way the docker daemon runs
OPTIONS=--insecure-registry controller:5050 --insecure-registry 192.168.1.254:5050 --bip=10.146.255.254/16 --fixed-cidr=10.146.255.255/16

# Location used for temporary files, such as those created by
# docker load and build operations. Default is /var/lib/docker/tmp
# Can be overriden by setting the following environment variable.
# DOCKER_TMPDIR=/var/tmp
```

**Comment by hansthen** [comment]
As I understand the error message, in Potsdam we had _only_  `--insecure-registry 192.168.1.254:5050`. As long as the cluster is in the office, we need a reference to both registries. So benchmark is correct. It is a bit unclean to leave the reference to the master after shipping the cluster to the customer's site, but it can do little harm.

**Comment by Marc69** [comment]
Both are there in
**Version:** r8 1b911a6
**Environment:** amaster

```
[root@controller-1(amaster*) ~]# cat /etc/sysconfig/docker
# /etc/sysconfig/docker

# Modify these options if you want to change the way the docker daemon runs
OPTIONS=--insecure-registry controller:5050 --insecure-registry 192.168.1.254:5050 --bip=10.146.255.254/16 --fixed-cidr=10.146.255.255/16 

# Location used for temporary files, such as those created by
# docker load and build operations. Default is /var/lib/docker/tmp
# Can be overriden by setting the following environment variable.
# DOCKER_TMPDIR=/var/tmp
```
